Topics/Tech_Stacks/Postman_Backend_Testing.md

```diff
@@ -0,0 +1,64 @@
+
+# Postman Backend Testing
+
+In this Postman application overview, we will go over how to 
+install the application and the details of how to tests your API endpoints.

```

Comment:
You only introduce "POST" and "GET" in the following article, you should probably write them here for clarity.